pkg/crypto/eddsa/babyjubjub.go

```diff
@@ -0,0 +1,69 @@
+package eddsa
```

Comment:
this package should not exist. As you can see these are utility functions of the store and should be used only once in the codebase. A code smell is that you are returning specific error types from this package.

Reply:
closing in favor of previous comment

Reply:
closing in favor of previous comment